ova/Libraries/provision-opendistro.sh

```diff
@@ -0,0 +1,293 @@
+
+
+        # # Configure JVM options for Elasticsearch
+        jv=$(java -version 2>&1 | grep -o -m1 '1.8.0') || : 
+        if [ "${jv}" = "1.8.0" ]; then
+            ln -s /usr/lib/jvm/java-1.8.0/lib/tools.jar /usr/share/elasticsearch/lib/
+            echo "root hard nproc 4096" >> /etc/security/limits.conf
+            echo "root soft nproc 4096" >> /etc/security/limits.conf
+            echo "elasticsearch hard nproc 4096" >> /etc/security/limits.conf
+            echo "elasticsearch soft nproc 4096" >> /etc/security/limits.conf
+            echo "bootstrap.system_call_filter: false" >> /etc/elasticsearch/elasticsearch.yml
+        fi
+        # Start Elasticsearch
+        startService "elasticsearch"
+        logger "Initializing Elasticsearch..."
+        until $(curl -XGET https://localhost:9200/ -uadmin:admin -k --max-time 120 --silent --output /dev/null); do
+            logger -ne $char
+            sleep 10
+        done
+
+        cd /usr/share/elasticsearch/plugins/opendistro_security/tools/ 
+        ./securityadmin.sh -cd ../securityconfig/ -nhnv -cacert /etc/elasticsearch/certs/root-ca.pem -cert /etc/elasticsearch/certs/admin.pem -key /etc/elasticsearch/certs/admin.key 
+
+        logger "Done"
+    fi
+}
+
+## Filebeat
+installFilebeat() {
+    logger "Installing Filebeat..."
+
+    $sys_type install filebeat-"${ELK_VERSION}" -y -q  
+    if [ "$?" != 0 ]; then
+        logger "Error: Filebeat installation failed"
+        exit 1;
+    else
+        WAZUH_MAJOR="$(echo ${WAZUH_VERSION} | head -c 1)"
+
+        curl -so /etc/filebeat/filebeat.yml ${resources_url}/resources/open-distro/filebeat/7.x/filebeat_all_in_one.yml --max-time 300  
+        curl -so /etc/filebeat/wazuh-template.json https://raw.githubusercontent.com/wazuh/wazuh/master/extensions/elasticsearch/7.x/wazuh-template.json --max-time 300 
+        chmod go+r /etc/filebeat/wazuh-template.json 
+        
+        curl -s https://packages.wazuh.com/3.x/filebeat/wazuh-filebeat-0.1.tar.gz --max-time 300 | tar -xvz -C /usr/share/filebeat/module 
+        mkdir -p /etc/filebeat/certs 
+        cp /etc/elasticsearch/certs/root-ca.pem /etc/filebeat/certs/ 
+        mv /etc/elasticsearch/certs/filebeat.key /etc/filebeat/certs/ 
+        mv /etc/elasticsearch/certs/filebeat.pem /etc/filebeat/certs/ 
+        # Start Filebeat
+        startService "filebeat"
+        logger "Done"
+    fi
+}
+
+## Kibana
+installKibana() {
+    WAZUH_MAJOR="$(echo ${WAZUH_VERSION} | head -c 1)"
+    logger "Installing Open Distro for Kibana..."
+    $sys_type install opendistroforelasticsearch-kibana-${OPENDISTRO_VERSION} -y -q 
+    if [ "$?" != 0 ]; then
+        logger "Error: Kibana installation failed"
+        exit 1;
+    else
+        curl -so /etc/kibana/kibana.yml ${resources_url}/resources/open-distro/kibana/7.x/kibana_all_in_one.yml --max-time 300 
+        echo "telemetry.enabled: false" >> /etc/kibana/kibana.yml
+        cd /usr/share/kibana 
```

Comment:
```suggestion
```

Reply:
Done in 599737656bcaaa119a2c30c76c1760fa0590643a